### DISCUSSION TITLE
☁ Add a new special cloud variable that pings a clouddata server for status! ☁

### DISCUSSION THREAD

**Discussion by ZekusV** [TurboWarp/packager#603] [Ideas]
I think it would be really cool to see a new special cloud variable called `☁ ping server` that pings a clouddata server whenever the value of the variable changes and the output for the server status can be in another special clouddata variable `☁ ping output` or something like so, this would be amazing to have and super useful for me and potentially a lot more people so that the people playing our projects know if they are actually connected to our own cloud servers or not, I really hope you do put this up for a feature, and would be super useful! Thank you for reading this. :)

**Comment by TheShovel** [comment]
Pretty sure you can do this with a custom extension or eval.

**Reply by ZekusV** [reply]
Which custom extensions may I ask?

**Reply by GarboMuffin** [reply]
I'm not aware of any custom extensions that would do this but custom (unsandboxed) extensions are preferred over special cloud behaviors moving forward, so this would probably be implemented as an extension

**Reply by GarboMuffin** [reply]
open an issue in https://github.com/TurboWarp/extensions/issues
